06_puppy/runnerdave/sync_store.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+// SyncStore implementation of the Storer interface
+type SyncStore struct {
+	sync.Mutex
```

Comment:
I am not experienced in using `sync.Mutex` but I think it is redundant in sync_store - otherwise, what's the point in using a `sync.Map` if we need to lock/unlock all the time?

Reply:
I am not that experienced either, but I am following the hint given at: https://github.com/anz-bank/go-samplerest/blob/master/pkg/pet/memstore.go#L8